script/tool/test/native_test_command_test.dart

```diff
@@ -53,6 +55,16 @@ final Map<String, dynamic> _kDeviceListMap = <String, dynamic>{
   }
 };
 
+const String _fakeCmakeCommand = 'path/to/cmake';
+
+void _createFakeCmakeCache(Directory pluginDir, Platform platform) {
```

Comment:
Inconsistent camel casing, above it is "CMakeProject" here it is "CmakeCache".

Reply:
Yep, I'll fix this. I keep messing this up because I'm mostly interacting with the `cmake` command so usually don't think about it being capitalized.

Reply:
Fixed; the only inconsistent version left should be where I'm specifically referring to the command `cmake`, where the lack of capitalization is deliberate.